src/sessions/PageManager.ts

```diff
@@ -156,10 +339,68 @@ export class PageManager {
         this.record(PAGE_VIEW_TYPE, this.createPageViewEvent());
     }
 
+    private moveItemsFromBuffer = (item: any) => {
+        this.page.ongoingActivity.add(item);
+    };
+
     /**
      * Returns true when cookies should be used to store user ID and session ID.
      */
     private useCookies() {
         return navigator.cookieEnabled && this.config.allowCookies;
     }
+
+    /**
+     * Checks whether the virtual page is still being loaded.
+     * If completed:
+     * (1) Clear the timers
+     * (2) Record data using NavigationEvent
+     * (3) Indicate current page has finished loading
+     */
+    private periodicCheckerCallback = () => {
```

Comment:
nit: Rename as verb of verb phrase (e.g., `checkLoadStatus`)

Reply:
Changed to `checkLoadStatus`